server/repository/src/migrations/templates/adding_table/mod.rs

```diff
@@ -0,0 +1,110 @@
+use crate::{migrations::*, StorageConnection};
+pub(crate) struct V1_00_05;
+
+impl Migration for V1_00_05 {
+    fn version(&self) -> Version {
+        Version::from_str("1.0.5")
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        // DATE, DATETIME, etc.. comes from src/migrations/types.rs, which is re-exported in src/migrations/mod.rs
+        sql!(
+            connection,
+            r#"
+        CREATE TABLE test_table (
+            id TEXT NOT NULL PRIMARY KEY,
+            -- Common types for postgres and sqlite
+            string_field TEXT,
+            not_null_string_field TEXT NOT NULL,
+            integer_field INTEGER,
+            big_integer_field BIGINT,
+            boolean_field BOOLEAN,
+            -- Types that differ between postgres and sqlite
+            date_field {DATE},
```

Comment:
only recently figure out that you can:

```rust
const yoweser = "yow".to_string();
println!("{yoweser} and debug version: {yoweser:?}")
```

the `sql!` uses `std::fmt` via `format!`